src/Provider/Yandex/Yandex.php

```diff
@@ -96,18 +104,22 @@ public function getName(): string
     }
 
     /**
-     * @param string      $url
-     * @param string|null $locale
-     * @param int         $limit
+     * @param string $url
+     * @param int    $limit
+     * @param string $locale
      *
      * @return AddressCollection
      */
-    private function executeQuery(string $url, string $locale = null, int $limit): AddressCollection
+    private function executeQuery(string $url, int $limit, string $locale = null): AddressCollection
```

Comment:
Why switch the order ?

Reply:
As per PSR2:

> Method arguments with default values MUST go at the end of the argument list.

https://www.php-fig.org/psr/psr-2/#44-method-arguments

It's only a change on the private API, so I don't think it's a problem

Reply:
Sorry, I indeed didn't see that one of the 2 had a default value.